### PR TITLE
chore!: Make eslint@8 the minimum supported version for this package

### PR DESCRIPTION
**Pull request by scagood**
See #25 #27

This simply bumps the lowest supported version of eslint to v8.

~TODO: Make the same change in the https://github.com/AtomLinter/linter-eslint~

### CODE REVIEW

**Comment by scagood** [comment]
I was wondering why the tests passed on my machine 🤔
I cant seem to run some of the tests locally 😢 

I will try to fix these tests without being able to run them!

```shell
[~/github/linter-eslint-node (minimum-version-8)] $ atom --test spec/package-interaction-spec.js
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/pathwatcher/build/Release/pathwatcher.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/oniguruma/build/Release/onig_scanner.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/superstring/build/Release/superstring.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/tree-sitter/build/Release/tree_sitter_runtime_binding.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/@atom/nsfw/build/Release/nsfw.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:82269) Electron: Loading non-context-aware native module in renderer: '/Applications/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/git-utils/build/Release/git.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
Installing ./atom
Installing ./attach-to-dom
Installing jasmine2-atom-matchers
Installing jasmine2-json
Installing jasmine-pass
Installing jasmine2-tagged
Installing jasmine2-focused
Installing jasmine-local-storage
Installing ./profile
Installing ./set
Installing jasmine-unspy
Installing ./ci
Started


No specs found
Finished in 0.012 seconds
Incomplete: No specs found
```

**Comment by scagood** [comment]
When checking on the linter-eslint repo I can see that there is no change needed in terms of the alert:
https://github.com/AtomLinter/linter-eslint/blob/master/src/worker-helpers.js#L138-L141

**Review by UziTech — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -15,19 +15,19 @@ The `linter` package will be installed for you if it’s not already present in
 
 Two reasons:
 
-1. After it was deprecated in v7, the `CLIEngine` class that `linter-eslint` relied upon was removed from ESLint in v8. Its replacement removed a few methods that supported some of `linter-eslint`’s features, making it impossible to abstract away the difference between the two, or to deliver an experience that’s consistent across ESLint versions.
+1. After it was deprecated in v8, the `CLIEngine` class that `linter-eslint` relied upon was removed from ESLint in v8. Its replacement removed a few methods that supported some of `linter-eslint`’s features, making it impossible to abstract away the difference between the two, or to deliver an experience that’s consistent across ESLint versions.
```
[the remark this answers is not in the document]
Comment (scagood) [reply README.md:18]: 😵‍💫 Good point!
```suggestion
1. After it was deprecated in v7, the `CLIEngine` class that `linter-eslint` relied upon was removed from ESLint in v8. Its replacement removed a few methods that supported some of `linter-eslint`’s features, making it impossible to abstract away the difference between the two, or to deliver an experience that’s consistent across ESLint versions.
```

**Comment by scagood** [comment]
Force push due to: https://github.com/AtomLinter/linter-eslint-node/runs/6786870142?check_suite_focus=true

**Comment by UziTech** [comment]
I usually stick with `chore(lint):` but it's not a big deal semantic release will see `lint:` and `chore(lint):` as the same.  CommitLint is more picky than it needs to be.

File: README.md [hunk 2]
```diff
@@ -21,13 +21,13 @@ Two reasons:
...
+If _all_ your projects use ESLint `>=8.0.0`, you can keep this package and uninstall `linter-eslint`. If _any_ of your projects use an older ESLint, you should keep `linter-eslint` installed alongside this package.
 
-Since they can both lint when ESLint 7.x is present, they have to coordinate who does the linting when both packages are installed. **If `linter-eslint` is installed, this package will not perform linting in ESLint 7.x environments** — only 8.x or greater. If only this package is installed, it will lint with any version of ESLint it supports.
+If only this package is installed, it will lint with any version of ESLint it supports.
 
-When `linter-eslint` is not installed and this package detects an ESLint version too old for it to support, it will show a notification and invite you to install `linter-eslint`. This behavior can be disabled in package settings.
+When `linter-eslint` is not installed and this package detects an ESLint version too old for it to support, it will show a notification and invite you to install `linter-eslint`. This behaviour can be disabled in package settings.
```
Comment (savetheclocktower) [inline README.md:30]: Oh, man, I just noticed this change. You trying to pick a fight? 😆

Comment (scagood) [reply README.md:30]: Here was me just removing the little red lines in my editor! 😂 🥊 😱

I can change it back 👀

Comment (savetheclocktower) [reply README.md:30]: Nah, I'll get it at some point, no worries

**Pull request merged**